epub34/a11y/index.html

```diff
@@ -1677,31 +1663,31 @@ <h2>Optimized publications</h2>
 			<h2>Distribution</h2>
 
 			<div class="note">
-				<p>Although <a href="https://www.w3.org/TR/epub/#dfn-epub-creator">EPUB creators</a> do not have to
-					follow the recommendations in this section to conform to this specification, some jurisdictions
-					require EPUB creators to follow similar practices. <a
+				<p>Although <a href="https://www.w3.org/TR/epub/#dfn-epub-publication">EPUB publications</a> do not have
+					to meet the recommendations in this section to conform to this specification, some jurisdictions
+					require similar distribution practices. <a
```

Comment:
This seems less clear to me. The previous text made it clear that some jurisdictions require creators to do certain things, this seems less clear that there may be additional legal requirements I have to consider. Maybe
```suggestion
					require EPUB publications to follow similar practices. <a
```

I dunno. Maybe that isn't better, but it seems clearer to me.